Ported all CNSFV codes from Bighorn to Navier-Stokes

yidongxiainl opened the pull request:
CNSFV stands for Compressible Navier-Stokes by Finite Volume

- Porting all CNSFV codes from Bighorn to Navier-Stokes module. They will serve as coding template for implementing other multi-dimensional convective flow and transport systems.

closes #8228

yidongxiainl committed:
Ported all CNSFV codes from Bighorn to Navier-Stokes

CNSFV stands for Compressible Navier-Stokes by Finite Volume

closes #8228

permcody commented:
Will this break Bighorn build?

permcody commented:
@andrsd, @jwpeterson - would you rather we import history?

yidongxiainl commented:
@permcody Yes and No. I have made relevant changes in Bighorn too, but not committed yet.

andrsd commented:
Since we test bighorn with the latest MOOSE, it will break.

We do not need to import the history, CNSFV is very young and 95% was done by @yidongxiainl

yidongxiainl commented:
@permcody This will again be a long effort to its final shape. I am doing early testing just because I want to eliminate bugs in the codes first. Then I will work on the new online documentation within this PR.

moosebuild commented:
Job `App tests:linux-gnu` on 89d04df : invalidated by @andrsd

bighorn should be fixed

yidongxiainl committed:
Built online doc navier_stokes module

Commit:
add .failed_tests to .gitignore

Add .failed_tests to the .gitignore file. Was an oversight that it was
not included in the PR responsible for implementing --failed-tests feature.

Refs #8509

moosebuild commented:
Job `Moose Documentation:linux-clang` on 1d6a1c4 wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/8344/site

moosebuild commented:
Job `Precheck:linux-gnu` on 1d6a1c4 : invalidated by @rwcarlsen

moosebuild commented:
Job `Moose Documentation:linux-clang` on e3b6dda wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/8344/site

andrsd commented:
@yidongxiainl You will need to add MOOSE module header to all `.h` and `.C` files

moosebuild commented:
Job `Moose Documentation:linux-clang` on 8fb1e80 wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/8344/site

moosebuild commented:
Job `Moose Documentation:linux-clang` on d885cd2 wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/8344/site

moosebuild commented:
Job `Moose Documentation:linux-clang` on 200b197 wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/8344/site

moosebuild commented:
Job `Moose Documentation:linux-clang` on d5d63ba wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/8344/site

yidongxiainl commented:
@andrsd Added MOOSE module header to all source code files. Passed all checks. Need peer review now.

andrsd commented on modules/navier_stokes/src/userobjects/CNSFVLeastSquaresSlopeReconstruction.C:
```diff
@@ -0,0 +1,367 @@
+void
+CNSFVLeastSquaresSlopeReconstruction::reconstructElementSlope()
+{
+  const Elem* elem = _current_elem;
```
Spaces around `*`

yidongxiainl replied:
Fixed.

andrsd commented on modules/navier_stokes/src/userobjects/CNSFVLeastSquaresSlopeReconstruction.C:
```diff
@@ -0,0 +1,367 @@
+
+//  if (bndElem)
+//    for (unsigned int iv = 0; iv < nvars; iv++)
+//      ugrad[iv] = 0.;
```
Do we need this?

yidongxiainl replied:
No. Removed.

andrsd commented on modules/navier_stokes/src/userobjects/CNSFVSlopeLimitingOneD.C:
```diff
@@ -0,0 +1,260 @@
+        }
+      }
+    }
+    break;
```
The proper indebtantion is:
```
case 0:
  code;
  more_code;
  break;
```

yidongxiainl replied:
Fixed.

andrsd commented on modules/navier_stokes/tests/cnsfv/1d_double_rarefaction_wave.i:
```diff
@@ -0,0 +1,253 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces...

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_bump_impl_ref2.i:
```diff
@@ -0,0 +1,418 @@
+[]
+############################################################
+[Functions]
+[]
```
No empty blocks...

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_bump_impl_ref3.i:
```diff
@@ -0,0 +1,418 @@
+  infinity_density = 1.
+  infinity_x_velocity = 0.5
+  infinity_pressure = 0.71428571428571428571
+  implicit = true
```
`implicit` is `true` by default, we do not need it...

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_bump_impl_ref3.i:
```diff
@@ -0,0 +1,418 @@
+[Kernels]
+  #### Time derivative of mass
+  [./time_rho]
+    implicit = true
```
This is most likely copy and paste from an explicit solve, `implicit = true` by default, no need for it.

yidongxiainl replied:
All fixed.

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_cyln_expl_lv3.i:
```diff
@@ -0,0 +1,366 @@
+  type = Transient
+  [./TimeIntegrator]
+    type = ExplicitEuler
+    #type = ExplicitTVDRK2
```
Remove...

yidongxiainl replied:
Done.

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_cyln_impl_lv2.i:
```diff
@@ -0,0 +1,371 @@
+  nl_abs_tol = 1e-8
+  nl_max_its = 10
+
+  #dt = 1.
```
Remove

yidongxiainl replied:
Done

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_obliqueshock_expl_weno.i:
```diff
@@ -0,0 +1,424 @@
+#    petsc_options_value = 'ilu'
+#    solve_type = 'NEWTON'
+#  [../]
+#[]
```
Remove this.

yidongxiainl replied:
Done.

yidongxiainl commented on modules/navier_stokes/include/bcs/CNSFVBC.h:
```diff
@@ -0,0 +1,87 @@
+ *
+ * Notes:
+ *
+ *   1. This BC kernel itself does not do any complex calculation.
```
Fixed

yidongxiainl commented on modules/navier_stokes/include/dgkernels/CNSFVKernel.h:
```diff
@@ -0,0 +1,96 @@
+ *      It gets the internal side flux vector and Jacobian matrix
+ *      from the internal side flux user object being called.
+ *
+ *   2. In general, a approximate Riemann solver should be used
```
Fixed

yidongxiainl commented on modules/navier_stokes/include/userobjects/CNSFVGreenGaussSlopeReconstruction.h:
```diff
@@ -0,0 +1,50 @@
+ * A user object that performs Green-Gauss slope reconstruction to get the slopes of the P0 primitive variables
+ */
+class CNSFVGreenGaussSlopeReconstruction :
+    public SlopeReconstructionMultiD
```
Fixed

yidongxiainl commented on modules/navier_stokes/src/auxkernels/CNSFVEntropyProductionAux.C:
```diff
@@ -0,0 +1,45 @@
+}
+
+CNSFVEntropyProductionAux::CNSFVEntropyProductionAux(const InputParameters & parameters):
+    AuxKernel(parameters),
```
All fixed.

yidongxiainl commented on modules/navier_stokes/src/auxkernels/CNSFVSpecificTotalEnthalpyAux.C:
```diff
@@ -0,0 +1,28 @@
+{
+     InputParameters params = validParams<AuxKernel>();
+     params.addClassDescription("An aux kernel for calculating specific total enthalpy.");
+     return params;
```
All fixed.

yidongxiainl commented on modules/navier_stokes/include/materials/CNSFVMaterial.h:
```diff
@@ -0,0 +1,84 @@
+
+  /// piecewise constant conserved variables
+
+  const VariableValue & _rhoc;  /// density
```
All fixed.

yidongxiainl commented on modules/navier_stokes/src/bcs/CNSFVBC.C:
```diff
@@ -0,0 +1,136 @@
+                             _rhou1[_qp],
+                             _rhov1[_qp],
+                             _rhow1[_qp],
+                             _rhoe1[_qp]};
```
All fixed.

yidongxiainl commented on modules/navier_stokes/src/dgkernels/CNSFVKernel.C:
```diff
@@ -0,0 +1,235 @@
+    DGKernel(parameters),
+    _component(getParam<MooseEnum>("component")),
+     _rhoc1(coupledValue("rho")),
+    _rhouc1(coupledValue("rhou")),
```
All fixed.

yidongxiainl commented on modules/navier_stokes/src/dgkernels/CNSFVKernel.C:
```diff
@@ -0,0 +1,235 @@
+  {
+    case Moose::Element:
+      re =  flux[_component] * _test[_i][_qp];
+    break;
```
Fixed.

yidongxiainl commented on modules/navier_stokes/src/postprocessors/CNSFVIdealGasEntropyL2Error.C:
```diff
@@ -0,0 +1,53 @@
+            * std::pow(_inf_rho / _rho[_qp], _fp.gamma(0., 0.))
+            - 1.;
+
+ return diff * diff;
```
Fixed.

yidongxiainl commented on modules/navier_stokes/src/userobjects/CNSFVFreeInflowBoundaryFlux.C:
```diff
@@ -0,0 +1,94 @@
+  //Real rhou1 = uvec1[1];
+  //Real rhov1 = uvec1[2];
+  //Real rhow1 = uvec1[3];
+  //Real rhoe1 = uvec1[4];
```
Yes. Need to denote the correct correspondence from uvec1

yidongxiainl commented on modules/navier_stokes/src/userobjects/CNSFVFreeOutflowBoundaryFlux.C:
```diff
@@ -0,0 +1,83 @@
+  Real e = rhoe1 / rho1 - 0.5 * (uadv1 * uadv1 + vadv1 * vadv1 + wadv1 * wadv1);
+  Real pres1 = _fp.pressure(v, e);
+  //Real csou1 = _fp.c(v, e);
+  //Real enth1 = (rhoe1 + pres1) / rho1;
```
No. Removed.

yidongxiainl commented on modules/navier_stokes/src/userobjects/CNSFVGreenGaussSlopeReconstruction.C:
```diff
@@ -0,0 +1,262 @@
+void
+CNSFVGreenGaussSlopeReconstruction::reconstructElementSlope()
+{
+  const Elem* elem = _current_elem;
```
Fixed.

yidongxiainl commented on modules/navier_stokes/src/userobjects/CNSFVHLLCInflowOutflowBoundaryFlux.C:
```diff
@@ -0,0 +1,827 @@
+    else
+    {
+      mooseError2("Weird wave speed occured in" , "\n"
+        , name() , ": " , __FUNCTION__ , "\n"
```
All fixed.

yidongxiainl commented on modules/navier_stokes/src/userobjects/CNSFVHLLCInflowOutflowBoundaryFlux.C:
```diff
@@ -0,0 +1,827 @@
+    , name() , ": " , __FUNCTION__ , "\n"
+    , "ielem = " , ielem , "\n"
+    , "iside = " , iside , "\n"
+    , "mach1 = " , mach1 , "\n");
```
All fixed.

yidongxiainl commented on docs/content/documentation/modules/navier_stokes/tmp.md:
```diff
@@ -0,0 +1,2 @@
+
+
```
No use. Will delete it.

moosebuild commented:
Job `Moose Documentation:linux-clang` on f039d73 wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/8344/site

moosebuild commented:
Job `Moose Documentation:linux-clang` on 7ef84b7 wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/8344/site

andrsd commented on modules/navier_stokes/src/auxkernels/CNSFVEntropyProductionAux.C:
```diff
@@ -0,0 +1,45 @@
+Real
+CNSFVEntropyProductionAux::computeValue()
+{
+    return (_pres[_qp] / _inf_pres)
```
2 spaces here...

andrsd commented on modules/navier_stokes/src/auxkernels/CNSFVSpecificTotalEnthalpyAux.C:
```diff
@@ -0,0 +1,28 @@
+Real
+CNSFVSpecificTotalEnthalpyAux::computeValue()
+{
+    return _enth[_qp];
```
2 spaces.

andrsd commented on modules/navier_stokes/src/postprocessors/CNSFVIdealGasTotalEnthalpyL2Error.C:
```diff
@@ -0,0 +1,69 @@
+                                _inf_vadv * _inf_vadv +
+                                _inf_wadv * _inf_wadv);
+
+ return diff * diff;
```
2 spaces.

andrsd commented on modules/navier_stokes/src/userobjects/CNSFVHLLCInternalSideFlux.C:
```diff
@@ -0,0 +1,833 @@
+
+  /// compute the S_M defined by Eq. (12).
+
+  Real sm = (rho2 * vdon2 * dsv2 - rho1 * vdon1 * dsv1 + pres1 - pres2)/
```
`) /` (missing space)

andrsd commented on modules/navier_stokes/src/userobjects/CNSFVHLLCInternalSideFlux.C:
```diff
@@ -0,0 +1,833 @@
+  Real rhour = omeg2 * (dsv2 * rhou2 + prst2 * nx);
+  Real rhovr = omeg2 * (dsv2 * rhov2 + prst2 * ny);
+  Real rhowr = omeg2 * (dsv2 * rhow2 + prst2 * nz);
+  Real rhoer = omeg2 * (dsv2 * rhoe2 - pres2 * vdon2 + prsta*sm);
```
spaces around `*`.

andrsd commented on modules/navier_stokes/src/userobjects/CNSFVHLLCInternalSideFlux.C:
```diff
@@ -0,0 +1,833 @@
+    Real rhour = omeg2 * (dsv2 * rhou2 + prst2 * nx);
+    Real rhovr = omeg2 * (dsv2 * rhov2 + prst2 * ny);
+    Real rhowr = omeg2 * (dsv2 * rhow2 + prst2 * nz);
+    Real rhoer = omeg2 * (dsv2 * rhoe2 - pres2 * vdon2 + prsta *sm );
```
space after `*`

andrsd commented on modules/navier_stokes/src/userobjects/CNSFVNoSlopeReconstruction.C:
```diff
@@ -0,0 +1,38 @@
+void
+CNSFVNoSlopeReconstruction::reconstructElementSlope()
+{
+  const Elem* elem = _current_elem;
```
space before `*`

andrsd commented on modules/navier_stokes/tests/cnsfv/1d_sedov_blast_wave.i:
```diff
@@ -0,0 +1,297 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/1d_sod_shock_tube.i:
```diff
@@ -0,0 +1,247 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/1d_square_wave_mc_limiter.i:
```diff
@@ -0,0 +1,257 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/1d_square_wave_minmod_limiter.i:
```diff
@@ -0,0 +1,257 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/1d_square_wave_superbee_limiter.i:
```diff
@@ -0,0 +1,257 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/1d_woodward_colella_blast_wave.i:
```diff
@@ -0,0 +1,260 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_bowshock_expl_lv0.i:
```diff
@@ -0,0 +1,376 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_bowshock_expl_lv0.i:
```diff
@@ -0,0 +1,376 @@
+[]
+############################################################
+[Functions]
+[]
```
Remove

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_bowshock_expl_lv1.i:
```diff
@@ -0,0 +1,376 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_bowshock_expl_lv1.i:
```diff
@@ -0,0 +1,376 @@
+[]
+############################################################
+[Functions]
+[]
```
Remove the empty block

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_bowshock_expl_lv2.i:
```diff
@@ -0,0 +1,376 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 space

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_bowshock_expl_lv2.i:
```diff
@@ -0,0 +1,376 @@
+[]
+############################################################
+[Functions]
+[]
```
Empty block -> remove.

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_bump_impl_ref1.i:
```diff
@@ -0,0 +1,413 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_bump_impl_ref1.i:
```diff
@@ -0,0 +1,413 @@
+[]
+############################################################
+[Functions]
+[]
```
Empty block -> remove.

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_bump_impl_ref3.i:
```diff
@@ -0,0 +1,413 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_bump_impl_ref3.i:
```diff
@@ -0,0 +1,413 @@
+[]
+############################################################
+[Functions]
+[]
```
Empty block -> remove.

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_bump_impl_ref4.i:
```diff
@@ -0,0 +1,413 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_bump_impl_ref4.i:
```diff
@@ -0,0 +1,413 @@
+[]
+############################################################
+[Functions]
+[]
```
Empty block -> remove

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_bump_impl_ref5.i:
```diff
@@ -0,0 +1,413 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_bump_impl_ref5.i:
```diff
@@ -0,0 +1,413 @@
+[]
+############################################################
+[Functions]
+[]
```
Empty block -> remove.

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_cyln_expl_lv1.i:
```diff
@@ -0,0 +1,365 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_cyln_expl_lv1.i:
```diff
@@ -0,0 +1,365 @@
+[]
+############################################################
+[Functions]
+[]
```
Empty block -> remove

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_cyln_expl_lv2.i:
```diff
@@ -0,0 +1,365 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_cyln_expl_lv2.i:
```diff
@@ -0,0 +1,365 @@
+[]
+############################################################
+[Functions]
+[]
```
Empty block -> remove

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_cyln_expl_lv3.i:
```diff
@@ -0,0 +1,365 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_cyln_expl_lv3.i:
```diff
@@ -0,0 +1,365 @@
+[]
+############################################################
+[Functions]
+[]
```
Empty block -> remove

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_cyln_expl_lv4.i:
```diff
@@ -0,0 +1,365 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_cyln_expl_lv4.i:
```diff
@@ -0,0 +1,365 @@
+[]
+############################################################
+[Functions]
+[]
```
Empty block -> remove

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_cyln_impl_lv1.i:
```diff
@@ -0,0 +1,366 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_cyln_impl_lv1.i:
```diff
@@ -0,0 +1,366 @@
+[]
+############################################################
+[Functions]
+[]
```
Empty block -> remove

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_cyln_impl_lv2.i:
```diff
@@ -0,0 +1,364 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_cyln_impl_lv2.i:
```diff
@@ -0,0 +1,364 @@
+[]
+############################################################
+[Functions]
+[]
```
Empty block -> remove

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_cyln_impl_lv3.i:
```diff
@@ -0,0 +1,366 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_cyln_impl_lv3.i:
```diff
@@ -0,0 +1,366 @@
+[]
+############################################################
+[Functions]
+[]
```
Empty block -> remove

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_cyln_impl_lv4.i:
```diff
@@ -0,0 +1,366 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_cyln_impl_lv4.i:
```diff
@@ -0,0 +1,366 @@
+[]
+############################################################
+[Functions]
+[]
```
Empty block -> remove

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_lax_shock_tube.i:
```diff
@@ -0,0 +1,302 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_mach3step.i:
```diff
@@ -0,0 +1,391 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_mach3step.i:
```diff
@@ -0,0 +1,391 @@
+[]
+############################################################
+[Functions]
+[]
```
Empty block -> remove

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_obliqueshock_expl_weno.i:
```diff
@@ -0,0 +1,414 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_obliqueshock_expl_weno.i:
```diff
@@ -0,0 +1,414 @@
+[]
+############################################################
+[Functions]
+[]
```
Empty block -> remove

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_sod_shock_tube.i:
```diff
@@ -0,0 +1,294 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_sod_shock_tube_parallel2.i:
```diff
@@ -0,0 +1,294 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_sod_shock_tube_parallel4.i:
```diff
@@ -0,0 +1,294 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_square_wave_gg_minmax.i:
```diff
@@ -0,0 +1,298 @@
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
```
2 spaces

andrsd commented on modules/navier_stokes/tests/cnsfv/2d_square_wave_gg_minmax.i:
```diff
@@ -0,0 +1,298 @@
+[]
+############################################################
+[Functions]
+[]
```
Empty block -> remove

moosebuild commented:
Job `Moose Documentation:linux-clang` on 31ff56a wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/8344/site

moosebuild commented:
Job `Moose Documentation:linux-clang` on 4c1bd23 wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/8344/site

yidongxiainl commented:
@permcody would you take an overview and see if there is anything noticeable that you would like me to revise?

moosebuild commented:
Job `Moose Documentation:linux-clang` on 7f324eb wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/8344/site

yidongxiainl commented:
@permcody @andrsd  One test failed in Test Heavy, which though is not in the navier_stokes module.

brianmoose commented:
The heavy test that failed is a known failure and has nothing to do with this PR.

brianmoose commented:
@yidongxiainl You have patches for wolverine and moltres right?

brianmoose commented:
This also broke the 64bit build https://www.moosebuild.org/job/68808/

andrsd commented:
I will take care of the 64 bit fix...

andrsd commented:
@brianmoose I do not think this broke the wolverine build. The missing symbols are coming from rDG module (but this PR does not touch that). It almost looks like there are some stale object files that the build system picks up. I verified that wolverine builds fine with current devel on my machine (Mac). Maybe `git clean` the build box and resubmit the test?

andrsd commented:
I used wrong version of MOOSE. Now, it does not build with the right version...

I will have a patch ready in a moment.
